perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -34,6 +35,11 @@
 flags.DEFINE_integer('openstack_volume_size', 20,
                      'Size of the volume (GB)')
 
+flags.DEFINE_enum('openstack_scheduler_policy', NONE,
```

Comment:
Instead of a string that says 'None', could you change this to the Python constant None? Then the NONE variable can be removed.